packages/amplify-frontend-javascript/lib/aws_exports.js.ejs

```diff
@@ -1,6 +1,6 @@
 // WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
 
-const awsmobile = <%- JSON.stringify(props.configOutput, null, 4) %>;
+const awsconfig = <%- JSON.stringify(props.configOutput, null, 4) %>;
```

Comment:
Why is this change needed?

Reply:
because the code is in the my web project. I think, the variable name should be more inclusive.

Reply:
Unfortunately a lot of our blogs and documentation refer to this variable name, so we can't make this change.

Reply:
I have reverted aws_exports.js.ejs.
Maybe later we make extensive change for variable names